lambda_functions/vetext_incoming_forwarder_lambda/vetext_incoming_forwarder_lambda.py

```diff
@@ -117,25 +161,26 @@ def process_body_from_alb_invocation(event):
 
     if not event_body_encoded:
         logger.info("event_body from alb record was not present")
-        logger.debug(event)
+        logger.info(event)
 
-    event_body_decoded = parse_qsl(b64decode(event_body_encoded).decode('utf-8'))
+    event_body_decoded = parse_qsl(base64.b64decode(event_body_encoded).decode("utf-8"))
     logger.debug("Decoded event body %s", event_body_decoded)
 
     event_body = dict(event_body_decoded)
     logger.debug("Converted body to dictionary: %s", event_body)
 
-    if 'AddOns' in event_body:
+    if "AddOns" in event_body:
         logger.info("AddOns present in event_body: %s", event_body["AddOns"])
-        del event_body['AddOns']
+        del event_body["AddOns"]
         logger.info("Removed AddOns from event_body")
```

Comment:
Clear the second logger and make this debug. it is never used and eats up log records.